PowerShellTools.HostService/ServiceManagement/Debugging/PowershellServiceHost.cs

```diff
@@ -230,6 +230,11 @@ public void PopRunspace()
             {
                 _callback.SetRemoteRunspace(false);
             }
+
+            if (_attaching)

```

Comment:
Again use this flag does not seem to be a reliable thing, anything in runspace we can use?